website/source/layouts/docs.erb

```diff
@@ -465,28 +465,48 @@
       </li>
 
       <li<%= sidebar_current("docs-connect") %>>
-        <a href="/docs/connect/index.html">Connect</a>
+        <a href="/docs/connect/index.html">Connect - Service Mesh</a>
         <ul class="nav">
           <li<%= sidebar_current("docs-connect-config") %>>
             <a href="/docs/connect/configuration.html">Configuration</a>
           </li>
+          <li<%= sidebar_current("docs-connect-observability") %>>
+            <a href="/docs/connect/observability.html">Observability</a>
+          </li>
+        </li>
+        <li<%= sidebar_current("docs-connect-intentions") %>>
+          <a href="/docs/connect/intentions.html">Intentions - Security</a>
```

Comment:
```suggestion
          <a href="/docs/connect/intentions.html">Intentions (Security)</a>
```
I'm not sure if that will look the best, but I think it helps distinguish between this and the "securing connect" category.